cluster/saltbase/salt/top.sls

```diff
@@ -26,7 +26,11 @@ base:
   {% endif %}
 {% endif %}
     - logrotate
+{% if grains['cloud'] is defined and grains.cloud == 'gce' %}
+    - supervisor
+{% else %}
     - monit
```

Comment:
I haven't had a chance to test other platforms, so I'd prefer to be cautious, I'll send a follow up PR after I test other platforms